src/main/scala/org/allenai/deep_qa/experiments/datasets/Science.scala

```diff
@@ -60,17 +75,55 @@ object ScienceDatasets {
     omnibusMtfGradeEightDevQuestionsWithBuscBackground
   ), "/efs/data/dlfa/processed/omnibus_4_8_dev/")
 
+  /////////////////////////////////////////////////////////////////////
+  // Diagram Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val diagramMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.ai2diagramSentences_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10)
   )
 
+  /////////////////////////////////////////////////////////////////////
+  // OpenQA Multiple True False
+  /////////////////////////////////////////////////////////////////////
+
   val openQaMtfQuestionsWithBuscBackground: JValue = makeBackgroundDataset(
     ScienceFiles.openQaAnimalAndGeneralScience_multipleTrueFalse_appendAnswer,
     "plain sentence",
     ScienceCorpora.buscElasticSearchIndex(10))
 
+  /////////////////////////////////////////////////////////////////////
+  // Omnibus QA
+  /////////////////////////////////////////////////////////////////////
+
+  /*
+   * Fourth Grade BUSC Background Files
+   */
+
+  val omnibusQaGradeFourTrainBuscBackgroundFile: JValue = JohannesDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourTrainSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  val omnibusQaGradeFourDevBuscBackgroundFile: JValue = JohannesDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourDevSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  val omnibusQaGradeFourTestBuscBackgroundFile: JValue = JohannesDatasets.makePassageBackgroundFile(
+    ScienceFiles.omnibusGradeFourTestSentences_questionAndAnswer,
+    "question and answer",
+    ScienceCorpora.buscElasticSearchIndex(3)
+  )
+
+  /*
+   * Fourth Grade Questions with BUSC Background (no multiple choice options)
```

Comment:
What classes are you asking about?

The difference between these and the "multiple true false" options is just how the question is encoded.  In the MTF case, you get four lines per question, with each line containing a question/answer pair.  In the QA version, you get a single line per question, with a column for the question and a column for the answer options.  I really dislike the naming convention we have in here, but I'm not sure how to make it better.  If you have suggestions, I'm happy to hear them.